lib/src/modules/conditional.dart

```diff
@@ -32,11 +30,16 @@ abstract class _Always extends Module with CustomSystemVerilog {
 
   final Uniquifier _portUniquifier = Uniquifier();
 
-  _Always(this.conditionals,
-      {Logic? reset, Map<Logic, Logic>? resetVal, super.name = 'always'}) {
+  /// Accepts [conditionals] and set maps receivers to output
+  ///
+  /// If [reset] is set then For a receiver assign default reset value
+  /// as `0` unless a reset value for is present [resetValues]
+  _Always(List<Conditional> conditionals,
```

Comment:
Is it possible to replace `List<Conditional> conditionals` with `this.conditionals`?